### ISSUE TITLE
Duplicated Actvities synchronize after reset fetch date

### ISSUE THREAD

**Issue by giupardeb** [Freeyourgadget/Gadgetbridge#1611]
#### Before reporting a bug, please confirm the following:
- [x] I have read the [wiki](https://github.com/Freeyourgadget/Gadgetbridge/wiki), and I didn't find a solution to my problem / an answer to my question.
- [x] I have searched the [issues](https://github.com/Freeyourgadget/Gadgetbridge/issues), and I didn't find a solution to my problem / an answer to my question.
- [x] If you upload an image or other content, please make sure you have read and understood the [github policies and terms of services](https://help.github.com/articles/github-terms-of-service/#1-responsibility-for-user-generated-content)

#### Your issue is:
When I reset fetch date, whenever I tap on button sync, the activities are synchronized one by one. Morover, GB syncronized duplicated activities, as shown in the video (at minute 1:25).

#### Your Gadgetbridge version is: 0.36.0

https://mega.nz/#!3ZAQ1SqL!8SdCp4iT-qcYHjWDiwjYAbQdmdP3zrYSa-CADEJ2l9g

**Comment by ashimokawa** [comment]
@giupardeb 
Yes we are aware of the issue. Resetting the date is primarily meant for debugging, but I agree that the behaviour is confusing and should be improved

**Comment by vanous** [comment]
Actually, I like it as it allows me to re-retrieve stuff . Perhaps lets hide it or mark "for debuging ..."?
